shared/git/new-repo/index.js

```diff
@@ -61,10 +61,11 @@ class NewRepo extends React.Component<Props, State> {
           alignItems: 'center',
           paddingLeft: globalMargins.small,
           paddingRight: globalMargins.small,
+          width: '100%',
         }}
       >
         <Avatar isTeam={true} teamname={item} size={16} style={{marginRight: globalMargins.tiny}} />
-        <Text type="Header">{item}</Text>
+        <Text type="Header" lineClamp={1}>{item}</Text>
```

Comment:
I was trying to get this to ellipsize the text, but I couldn't get it to work. Any suggestions?

Reply:
look at the inbox row where we do this. the width needs to be explicit and you need to set some css